lessons/Algebra2/exploring-linear-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,265 @@
+
+@clear
+
+
+=== Investigate
+
+Now that you've dug into the role the form, direction and strength play in quantifying a correlation, it's time to put those concepts to work!
+
+[.lesson-instruction]
+In pairs or small groups, complete @printable-exercise{pages/identifying-form-matching.adoc}.
+
+Review student answers, and have students _explain their thinking_ for this activity. For students who are struggling, hearing what their peers are looking for is especially helpful at this stage.
+
+[.lesson-instruction]
+In pairs or small groups, complete @printable-exercise{pages/identifying-form.adoc}
+
+Review student answers. Some of the answers are not so clear-cut, and students may disagree about what constitutes a "strong" vs. "weak" correlation. We've tried to choose scatter plots that clearly fall into one category or the other, but without diving into the algorithm for linear regression students may find this exercise somewhat subjective.
+
+[.lesson-instruction]
+Return to @printable-exercise{exploring-states.adoc}, and complete the rest of the page.
+
+
+=== Common Misconceptions
+- Students often conflate strength and direction, thinking that a strong correlation _must_ be positive and a weak one _must_ be negative.
+- Students may also falsely believe that there is ALWAYS a correlation between any two variables in their dataset.
+- Students often believe that strength and sample size are interchangeable, leading to mistaken assumptions like "any correlation found in a million data points _must_ be strong!"
+
+
+=== Synthesize
+
+- What relationships did you find in the states dataset?
+- What was their form, direction, and strength?
+- Were any of these relationships a surprise? Why or why not?
+
+
+== Fitting Linear Models @duration{25 minutes}
+
+=== Overview
+
+The line of best fit is framed as a _predictor function_, which attempts to predict where a new point would fall on the plane based on the relationship in the data. Students define their predictors (linear functions) to find the line of best fit, using @vocab{R-squared} to determine fitness and making predictions with the result. They explore the impact that slope and and y-intercept have on fitness.
+
+=== Launch
+
+[.lesson-instruction]
+In Pyret, make a scatter plot showing the the relationship between `college-or-higher` and `median-household-income`.
+
+@center{@image{images/college-v-income.png}}
+
+This scatter plot appears to show a positive, linear relationship: states with higher percentages of college graduates tend to have higher median household incomes.
+
+[.lesson-instruction]
+Suppose the United States were to add a new state. **Based on the data for the existing 50 states, what median household income would you predict, if exactly 50% of the new state's citizens had attended college?** What about 90%? 10%
+
```

Comment:
The domain of the data shown in the scatterplot is between 20 and 50%. Are you sure we want to ask about 90% and 10%? Guessing from lesson plan that you assumed they were actually visible and it would make more sense to ask about 25% and 40% as other points.  But, if you stick to 10% and 90%, we need to provide more guidance for the discussion..

Reply:
Good point. Fixed.